tests/api/test_dedupe_router.py

```
@@ -178,6 +179,41 @@ def test_scan_with_options(self, mock_detector_cls, tmp_path: Path) -> None:
 class TestPreviewDuplicates:
     """Tests for POST /api/v1/dedupe/preview."""
 
+    def test_preview_keeps_oldest_file_by_modified_time(self) -> None:
+        older = datetime(2024, 1, 1, tzinfo=UTC)
+        newer = datetime(2024, 1, 2, tzinfo=UTC)
+        group = DedupeGroup(
+            hash_value="abc123",
+            files=[
+                DedupeFileInfo(path="/tmp/newer.txt", size=100, modified=newer, accessed=newer),
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/newer.txt"

(S108)